hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/request/DefaultEntityRequestBuilder.java

```diff
@@ -144,6 +169,11 @@ private TimeRangeArgument timeRange(Map<String, Object> arguments) {
     return this.argumentDeserializer.deserializePrimitive(arguments, SpaceArgument.class);
   }
 
+  private Optional<LabelRequest> buildLabelRequest(
```

Comment:
would suggest pulling this outside - a boolean that turns into an optional empty could just as easily (and with more clarity) be done above.